requirements-dev.txt

```diff
@@ -6,3 +6,4 @@ pytest-aiohttp==0.1.3
 pytest-asyncio==0.8.0
 pytest-mock==1.6.3
 tox==2.9.1
+cryptography==2.1.3
```

Comment:
it's no dev package, lets move to requirements.txt, also please take natural name ordering in file